esp-hal/src/soc/esp32c6/trng.rs

```diff
@@ -0,0 +1,440 @@
+const I2C_SAR_ADC: u8 = 0x69;
+const I2C_SAR_ADC_HOSTID: u8 = 0;
+
+const SAR2_CHANNEL: u32 = 9;
+const SAR2_ATTEN: u32 = 1;
+const SAR1_ATTEN: u32 = 1;
+const PATTERN_BIT_WIDTH: u32 = 6;
+
+const REGI2C_BBPLL: u8 = 0x66;
+const REGI2C_BIAS: u8 = 0x6a;
+const REGI2C_DIG_REG: u8 = 0x6d;
+const REGI2C_ULP_CAL: u8 = 0x61;
+const REGI2C_SAR_I2C: u8 = 0x69;
+
+const REGI2C_BBPLL_DEVICE_EN: u32 = 1 << 5;
```

Comment:
one day we proably should de-duplicate these definitions (we have some of them also in clocks_ll) - but definitely not in this PR

Reply:
Yep, we should do something about them `regi2c_*` functions through the whole codebase, definitely.